pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -129,11 +129,13 @@ public void revokePermissionsOnTopic(@PathParam("tenant") String tenant,
     @ApiResponses(value = {
         @ApiResponse(code = 403, message = "Don't have admin permission"),
         @ApiResponse(code = 409, message = "Partitioned topic already exist"),
-        @ApiResponse(code = 412, message = "Partitioned topic name is invalid")
+        @ApiResponse(code = 412, message = "Partitioned topic name is invalid"),
+            @ApiResponse(code = 24, message = "Namespace does not have any clusters configured")
```

Comment:
Seems the spaces alignment is not handled well here.

Reply:
will fix this indent